conductor_api/src/conductor/admin.rs

```diff
@@ -8,7 +8,11 @@ use crate::{
 };
 use holochain_core_types::{cas::content::AddressableContent, error::HolochainError};
 use json_patch;
-use std::{fs, path::PathBuf, sync::Arc};
+use std::{
+    fs,
+    path::PathBuf,
+    sync::{Arc, RwLock},
+};
 
 pub trait ConductorAdmin {
     fn install_dna_from_file(
```

Comment:
I would like to see another boolean parameter that declares if the instance should be started or not. I am not sure if starting it every time is the right thing to do

Reply:
Actually I need to rename the PR, because now this is not starting the instance, but only *instantiating* it, so it can be selectively started later.

Reply:
Oh right! I didn't know they were different things. So does it makes sense to hold an instantiated instance? probably not especially if they are all instantiated on reload anyway